### PR TITLE
forumdirectory: use the directory template from friendica core

### PR DESCRIPTION
**Pull request by rabuzarus**
The forumdirectory addon will use the directory template from core. So the addon is better maintainable and the GUI is familiar. In addition it fixes https://github.com/friendica/friendica/issues/6674

Before merging this PR https://github.com/friendica/friendica/pull/6683 needs to be accepted

NOTE:
the forum directory addon does use nearly the exact code of friendicas directory addon. In the future we should integrate the forumdirectory addon into the directory module by adding a `forum` parameter.
Hubzilla has a nice slider to switch between regular and forum account in the directory. We should try to adopt it (https://hub.libranet.de/directory)

### CODE REVIEW

File: forumdirectory/forumdirectory.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /**
  * Name: Forum Directory
  * Description: Add a directory of forums hosted on your server, with verbose descriptions.
- * Version: 1.0
+ * Version: 1.1
  * Author: Thomas Willingham <https://beardyunixer.com/profile/beardyunixer>
  */
```
Comment (MrPetovan) [inline forumdirectory/forumdirectory.php]: You can add yourself to the list of authors if you want.

File: forumdirectory/forumdirectory.php [hunk 2]
```diff
@@ -56,6 +57,7 @@ function forumdirectory_post(App $a)
 	if (!empty($_POST['search'])) {
 		$a->data['search'] = $_POST['search'];
 	}
+	Logger::log(print_r($_POST, true));
```
Comment (MrPetovan) [inline forumdirectory/forumdirectory.php]: Please use one of the new `Logger::(error|warning|notice|info|debug)` methods for new logger calls. This looks like a `Logger::debug()` call.

Comment (rabuzarus) [reply forumdirectory/forumdirectory.php]: This would be a `Logger::data()` call :-) But it shouldn't be there. I forgot to remove it

Comment (MrPetovan) [reply forumdirectory/forumdirectory.php]: There's no `DATA` nor `TRACE` level anymore, the most verbose is `debug` according to [PSR-3](https://www.php-fig.org/psr/psr-3/)

Comment (tobiasd) [reply forumdirectory/forumdirectory.php]: BTW is it somewhere documented which old level corresponds to which  new level?

File: forumdirectory/forumdirectory.php (hunk 1 above)
Comment (rabuzarus) [reply forumdirectory/forumdirectory.php]: I don't want to :-)

File: forumdirectory/forumdirectory.php (hunk 2 above)
Comment (MrPetovan) [reply forumdirectory/forumdirectory.php]: Yes sir:
https://github.com/friendica/friendica/blob/a3152a96c2d1b473f806e287d2e06b2ad4e165ed/src/Core/Logger.php#L16-L39

On the left are the legacy logger levels, on the right are the corresponding PSR-3 compliant levels.

**Pull request merged**